rest-api/alembic/versions/8786f0a95801_org_metadata.py

```diff
@@ -0,0 +1,84 @@
+"""Org metadata
+
+Revision ID: 8786f0a95801
+Revises: b315fec9aa4e
+Create Date: 2017-11-29 16:20:00.427120
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import model.utils
+from sqlalchemy.dialects import mysql
+
+from participant_enums import PhysicalMeasurementsStatus, QuestionnaireStatus, OrderStatus
+from participant_enums import WithdrawalStatus, SuspensionStatus
+from participant_enums import EnrollmentStatus, Race, SampleStatus, OrganizationType
+from model.site_enums import SiteStatus
+from model.code import CodeType
+
+# revision identifiers, used by Alembic.
+revision = '8786f0a95801'
+down_revision = 'b315fec9aa4e'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('organization',
+    sa.Column('organization_id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(length=80), nullable=False),
+    sa.Column('display_name', sa.String(length=255), nullable=False),
+    sa.Column('hpo_id', sa.Integer(), nullable=False),
+    sa.ForeignKeyConstraint(['hpo_id'], ['hpo.hpo_id'], ),
+    sa.PrimaryKeyConstraint('organization_id')
+    )
+    op.add_column(u'site', sa.Column('address_1', sa.String(length=1024), nullable=True))
+    op.add_column(u'site', sa.Column('address_2', sa.String(length=1024), nullable=True))
+    op.add_column(u'site', sa.Column('admin_emails', sa.String(length=4096), nullable=True))
```

Comment:
nit: some of these aren't in the design doc

Reply:
Updated the design doc to try to match this.

Let me know if you spot any differences.

As usually, docs go stale the minute you start coding. :)